mindmeld/auto_annotator.py

```diff
@@ -463,14 +477,53 @@ def _load_model(model):
 
     @property
     def supported_entity_types(self):  # pylint: disable=W0236
-        """
+        """This function generates a list of supported entities for the given language.
+        These entities labels are mapped to MindMeld sys_entities.
+        The "misc" spacy entity is skipped since the category too broad to be
+        helpful in an application.
+
         Returns:
             supported_entity_types (list): List of supported entity types.
         """
-        return SPACY_ANNOTATOR_SUPPORTED_ENTITIES
+        spacy_supported_entities = [e.lower() for e in self.nlp.get_pipe("ner").labels]
+        supported_entities = set()
+        for entity in spacy_supported_entities:
+            if entity == "misc":
+                continue
+            if entity in ["time", "date", "datetime"]:
```

Comment:
Is this not in the ANNOTATOR_TO_SYS_ENTITY_MAPPINGS dict because the same entity can map to different system entities?